src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionTestResultViewModelTests.swift

```diff
@@ -23,653 +23,659 @@ import XCTest
 // swiftlint:disable file_length
 // swiftlint:disable:next type_body_length
 class ExposureSubmissionTestResultViewModelTests: XCTestCase {
-
+	
```

Comment:
Use`MockTestStore` and add linebreaks after parameters in this file as well

Reply:
Done